src/ui/sass/modules/_SearchBar.scss

```diff
@@ -270,10 +270,10 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
     }
   }
 
-  &-buttonImage .Icon svg, .Icon-image
+  &-AnimatedIcon svg, &-buttonImage .Icon svg, .Icon-image
```

Comment:
This looks like it's shrinking the regular logo on percy since this styling affects it too